.env.production

```diff
@@ -0,0 +1,2 @@
+INLINE_RUNTIME_CHUNK=false
```

Comment:
What is this configuration?

Reply:
It generates a file for the runtime in place of including it in the index.html

Reply:
Correct.